packages/transactions/src/coders/transaction.ts

```diff
@@ -669,13 +580,79 @@ export class TransactionBlobCoder extends Coder<TransactionBlob, TransactionBlob
   }
 }
 
+export type TransactionUnknown = BaseTransactionType & {
+  type: TransactionType.Unknown;
+
+  /** Data of the transaction */
+  data?: BytesLike;
+};
+
+export class TransactionUnknownCoder extends Coder<TransactionUnknown, TransactionUnknown> {
```

Comment:
Given we don't know the spec of an unknown transaction and won't be transmitting it, why would we want to encode/decode it?

Reply:
Good point 6b88425ac